### PR TITLE
Add Early Access Program to issue template

### PR DESCRIPTION
**Pull request by mason-at-pieces**

### CODE REVIEW

**Review by shivay-at-pieces — APPROVED** [review]

File: .github/labeler/index.js [hunk 1]
```diff
@@ -30,6 +30,11 @@ async function run() {
   if (issueBodyLines[4] === '### Operating System') os = issueBodyLines[6].toLowerCase();
   if (os) labels.push(`os:${os}`);
 
+  // Check if Early Access Program checkbox is checked
+  if (issueBody.includes('- [x] Yes, this is related to an Early Access Program feature.')) {
+    labels.push('Early Access Program');
```
Comment (caleb-at-pieces) [inline .github/labeler/index.js:35]: i think our labels are all lowercase so maybe want this one to follow suit?

also i would consider just making it `eap`

**Review by caleb-at-pieces — COMMENTED** [review]
just double check the label here, I think all the other labels are lowercase

File: .github/labeler/index.js (hunk 1 above)
Comment (mason-at-pieces) [reply .github/labeler/index.js]: updated

Comment (mason-at-pieces) [reply .github/labeler/index.js]: Just made it `early access program`

**Pull request merged**